Dockerfile

```diff
@@ -26,6 +26,7 @@ ENV LANG=C.UTF-8
 # TODO: Do we want to create editable installs of plugins as well?  We
 # will need a plugin only requirements file for this.
 RUN pip install --upgrade --upgrade-strategy eager --editable .
+RUN pip install --upgrade --upgrade-strategy eager --editable ./plugins/oauth
```

Comment:
This should probably be done in a downstream image rather than here, since none of the other plugins are getting installed in this image.

Reply:
I will update the PR with a parametrized version of the Dockerfile, allowing users to build with whichever plugins they choose (none by default).

Reply:
I'd like to see what this sort of parameterization looks like, but for now, I think we should keep Girder's primary Docker image free of plugins. Downstreams can easily inherit from this image and install additional plugins if they wish.

Reply:
> Downstreams can easily inherit from this image and install additional plugins if they wish.

Looking at the original [Dockerfile](https://github.com/girder/girder/blob/master/Dockerfile) it looks like one has to install plugins (pip install ...) before calling 'girder build'.  If the plugins could be added afterwards, that's great:  We could have a Dockerfile starting with 'FROM girder/girder' which adds plugins.  But currently it's not clear how that would work.  Can you show how to do that?

Reply:
Indeed, downstreams would have something along the lines of:
```
FROM girder/girder
RUN pip install ./plugins/oauth \
  && girder build
```
I agree we should have better support and documentation for this use case though.